Makefile

```diff
@@ -13,4 +14,10 @@ formatSwift:
 
 formatDart:
 	dart format .
+
+checkFormatDart:
+	dart format --set-exit-if-changed ./
+
+# TODO: add analyze steps in CI
```

Comment:
failing for some reason, I guess different versions on my local machine and CI but everything is green locally, will check later